static/js/message_viewport.js

```diff
@@ -305,9 +306,8 @@ exports.stop_auto_scrolling = function () {
 exports.is_narrow = function () {
     // This basically returns true when we hide the right sidebar for
     // the left_side_userlist skinny mode.  It would be nice to have a less brittle
-    // test for this.  See the "@media (max-width: 1165px)" section in
-    // media.css.
-    return window.innerWidth <= 1165;
+    // test for this.
+    return window.innerWidth <= media_breakpoints["xl-max"];
```

Comment:
Stylistically, should this be `media_breakpoints.xl_max`?  While that is a bit less consistent with Bootstrap's naming, it would also mean that there will be type-checking when we move to typescript.

@andersk FYI for thoughts on this tradeoff.

Reply:
(I suppose we could just name our CSS variables e.g. `xl_max` rather than `xl-max` to avoid an inconsistency issue in our codebase; there's no reason the names have to exactly match Bootstrap)

Reply:
Okay used underscore everywhere.